librz/core/cmd_meta.c

```diff
@@ -472,14 +472,15 @@ static int cmd_meta_others(RzCore *core, const char *input) {
 			ut64 size;
 			RzAnalysisMetaItem *mi = rz_meta_get_at(core->analysis, addr, type, &size);
 			if (mi) {
-				rz_meta_print(core->analysis, mi, addr, size, input[3], NULL, false);
+				RzOutputMode mode = rz_char_to_output_mode3(&input[3]);
```

Comment:
When I use `rz_char_to_output_mode3` or `rz_char_to_output_mode` the tests fail. But when I use `rz_char_to_output_mode2` over here the tests don't fail. I haven't understood why this is the case especially when it looks like the implementation of all three functions are very similar.

Reply:
I'd start by addressing the issues I told you about not using `if(mode)`... becasue that, right now, just means `if (mode != RZ_OUTPUT_MODE_UNKNOWN)` and with `rz_char_to_output_mode` you may get `RZ_OUTPUT_MODE_UNKNOWN` indeed...

Those `if(meta)` should become something else.

Reply:
I had actually started with that but the number of errors started increasing. So I thought that was the wrong approach. Anyway, I'll try it again.

Reply:
> I'd start by addressing the issues I told you about not using if(mode)... becasue that, right now, just means if (mode != RZ_OUTPUT_MODE_UNKNOWN) and with rz_char_to_output_mode you may get RZ_OUTPUT_MODE_UNKNOWN indeed...

When i say that `if (mode)` right now just means `if(mode != RZ_OUTPUT_MODE_UNKNOWN)`, please be aware that probably that is not the meaning that it *should have* or that it had before your PR.